Source/WebCore/animation/AnimationTimeline.h

```diff
@@ -36,6 +36,24 @@ namespace WebCore {
 
 class WebAnimation;
 
+struct TimelineScope {
```

Comment:
Maybe nest this within `AnimationTimeline`?

Reply:
I think generally we don't nest these structs used for parsing so we don't need to prefix them everywhere.

Reply:
There is precedent at least in the animation world. Take a look at `Animation.h` and notice `TransitionProperty` and `Direction` for instance. Not a big deal though.

Reply:
That said, if you're going to keep it outside the scope of `AnimationTimeline`, might be better to have it be its own `AnimationTimelineScope.h` file.

Reply:
Separated this to its own file.